Patches/Rimeffect Core/Armor_ME.xml

```diff
@@ -0,0 +1,504 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+			<li>Rim-Effect: Core</li>
+		</mods>
+		<match Class="PatchOperationSequence">
+			<operations>
+
+
+				<li Class="PatchOperationAdd">
+					<xpath>Defs/ThingDef[defName="RE_Apparel_LightAllianceArmor" or defName="RE_Apparel_AllianceArmor" or defName="RE_Apparel_HeavyAllianceArmor" or defName="RE_Apparel_SpectreArmor"]/apparel/layers</xpath>
+					<value>
+						<li>Webbing</li>
```

Comment:
Existing power armor doesn't occupy these slots, so I don't know that this armor should.

Reply:
I'm doing this as I don't think Mass effect armour should not suffer the fate of being visually ruined by backpacks and tactical rigging.
Do you remember any ugly cloth pouches on the armour of anyone in any mass effect game? me neither.

Reply:
Well, then I'd give the armor gives sufficient carryBulk to offset the lack of webbing.

Reply:
Most armours already give sufficiently extra carrybulk so I'm not sure what's the issue here. Tacvest+Backpack is 30+20, but I'll add a bit of extra carry bulk anyway.

Reply:
I don't really plan to giving them too much ammo, ending up with an absolute ton of ammo after every raid is both annoying clutter and makes special ammo too easy to acquire.